js/main.js

```diff
@@ -0,0 +1,70 @@
+
+
+var myButton = document.querySelector("#donateBike");
+
+myButton.addEventListener("click", doSomething);
+
+function doSomething() {
+    var box = document.querySelector(".jumbotron");
+    if (!box.className.includes("red")) {
+        box.className += " red";
+    }
+}
+
+var myButton2 = document.querySelector("#addToLearnMore"); // Element that is going to call the action
+
+myButton2.addEventListener("click", doSomething2);
+
+function doSomething2() {
+    var box = document.querySelector("#mainArticles"); // Call the element that want to change
+    var paragraph = document.createElement("article.article");
+    paragraph.style.color = "blue";
+    paragraph.innerText = "SHOW ME MORE JAVASCRIPT!!";
+
+    box.appendChild(paragraph);
+}
+
+var images = document.querySelector("#allImages"); // Element that is going to call the action
+images.addEventListener("click", selectAll);
+
+function selectAll() {
+    var allImages = document.querySelectorAll("img");
+    for (var i = 0; i < allImages.length; i++) {
+        allImages[i].width = allImages[i].width - 10;
+        allImages[i].height = allImages[i].height - 10;
+
+    }
+};
+
+var divs = document.querySelector("#addToLearnMore"); // Element that is going to call the action
+divs.addEventListener("click", selectDivs);
+
+function selectDivs() {
+    var allDivs = document.querySelectorAll("div");
+    if (!allDivs.className.includes("pink")) {
+        allDivs.className += " pink";
+    }
+}
+
+// AJAX
+//creating a request object
+
+var addInfo = document.querySelector("#addArticleBtn"); // Element that is going to call the action
+addInfo.addEventListener("click", request);
```

Comment:
I suspect this isn't really working. If you want to add an action to the button you will need to add this to a function and then call the "send" into it. 

You can have the request object in the object, but the "request.send()" will only be executed once the file is read for the 1st time.